node/packages/aws-lambda-otel-extension/test/fixtures/lambdas/requires.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+// eslint-disable-next-line import/no-unresolved
+require('aws-sdk');
```

Comment:
It might make sense to require more than one supported library here that way we can see how the benchmarks are effected when loading multiple libraries.

If we include `apollo-server-lambda` this would also also include the graphql and express instrumentations and might make this app a bit closer to a real use case. In fact, if you would like I could turn this file into a small apollo server if you think that would be helpful?